starfish/models/surfer_model.py

```diff
@@ -26,8 +32,8 @@ def __init__(self, ocean, did=None, ddo=None, options=None):
 
         self._headers = {'content-type': 'application/json'}
         authorization = options.get('authorization')
-        if authorization:
-            self._headers['Authorization'] = f'Basic {authorization}'
+        if authorization: # this is an OAuth2 token
```

Comment:
Is there expected to be multiple types  of authorization (basic and/or OAuth2)? might need to think of adding an authorization  `type` to this.